src/compiler/base/compile/compile-child-scenarios-or-expects.xsl

```diff
@@ -39,12 +34,10 @@
       on-no-match="deep-skip" />
 
    <!--
-      At x:pending elements, we switch the $reason-for-pending tunnel param value for children.
+      Unwrap x:pending.
```

Comment:
The wording of the comment confused me at first because `x:pending` is the wrapper rather than the wrapped. I think you're trying to explain why all you have to do here is move on to the children. How about something like "Pending status and reason are accounted for in descendant context"? Same question for the file `src/compiler/base/invoke-compiled/invoke-compiled-child-scenarios-or-expects.xsl`.

Reply:
Thanks @galtm. Revised the comments in 20545e6cc2514efdbf36d1a12e4929fcb6bc6fe0.